about.html

```diff
@@ -0,0 +1,160 @@
+            </p>
+          </div>
+        </div>
+        <div class="reach-out">
+          <div class="paragraph-reach-out">
+            <p>Please you reach out to us via Email incase you want to find out more about DMT summit 2019!</p>
+          </div>
+          <p>taiwoenochdev@gmail.com</p>
+
+          <div class="bottom-line"></div>
+
+        </div>
+      </section>
+      <section>
+        <div class="summit-div">
+          <div class="creative">
+            <p>Danmert Nigeria Enterprise Summit 2019 Logo</p>
+          </div>
+
+          <div class="bottom-line-2"></div>
+
+          <div class="logo-description">
+            <p>The logo of DMT Summit was decided through the logo competition from 8. june to 7. july.</p>
+          </div>
+          <div>
+            <img src="./images/main-about-logo.png" alt="summit-logo"  class="summit-logo">
+          </div>
+        </div>
+        <div class="towards-end">
+          <div class="about-summit">
+            <p>See the past DMT Summits</p>
+
+            <div class="bottom-line-2"></div>
+
+          </div>
+          <div class="towards-end-paragraph">
+            <p>Take a look at last two Danmert Enterprise summit that took place in Nigeria</p>
+          </div>
+          <div class="need-flex">
+            <div class="summit-2013">
+              <p>2018 DMT Submit</p>
+            </div>
+            <div class="summit-2014">
+              <p>2017 DMT Submit</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </header>
+    <section>
+      <div class="site-icons">
+        <div>
+          <p class="partner">Partner</p>
+        </div>
+
+        <div class="border"></div>
+
+        <div class="bottom-icons">
+          <div>
+            <img src="./images/mozilla.png" alt="mozilla-icon" class="partner-1">
+          </div>
+          <div>
+            <img src="./images/google.png" alt="google-icon" class="partner-1">
+          </div>
+          <div>
+            <img src="./images/naver.png" alt="naver-icon" class="partner-1">
+          </div>
+          <div class="bottom-icon-2">
+            <div>
+              <img src="./images/kakao1.png" alt="kakao-icon" class="partner-1">
+            </div>
+            <div>
+              <img src="./images/airbnb.png" alt="airbnb-icon" class="partner-1">
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="main-footer">
+      <div class="footer-comment">
+        <div>
+          <img src="./images/logo-dmt_2.png" alt="company-logo" class="footer-logo">
+        </div>
+        <div>
+          <p>2019 Danmert NigeriaEnterprise reserve some comment</p>
+        </div>
+      </div>
+    </footer>
+    <script src="main.js"></script> -->
```

Comment:
- [ ] Kindly remove the `-->` arrow. It shows on the HTML page for the About page

![image](https://user-images.githubusercontent.com/77106962/218118886-66af28a3-e2f1-4bf6-ae0c-3c548fd9ca52.png)

Reply:
Done